core-bundle/src/Image/PictureFactoryInterface.php

```diff
@@ -31,7 +31,7 @@ public function setDefaultDensities($densities);
      * Creates a Picture object.
      *
      * @param string|ImageInterface               $path
-     * @param int|array|PictureConfiguration|null $size
+     * @param int|string|array|PictureConfiguration|null $size
```

Comment:
`int|string|array|PictureConfiguration|null` – does this not qualify as `mixed` already? 😄 @ausi /cc

Reply:
I’d prefer `int|string|array|PictureConfiguration|null` over `mixed`.

Reply:
Me2. I read the type hints as part of the documentation and this clarifies how the method can be used.